docs/application/web/guides/security/privacy-related-permissions.md

```diff
@@ -31,7 +31,7 @@ To check whether an application has permission to use a privilege, and to reques
     var result = tizen.ppm.checkPermission(cameraPrivilege);
```

Comment:
Line 27 and 25:  Please change as:
To verify whether

Reply:
Done